python/lsst/rubintv/production/monitorPlotting.py

```diff
@@ -62,4 +65,5 @@ def plotExp(exp, figure, saveFilename, scalingOption="default"):
     cax = divider.append_axes("right", size="5%", pad=0.05)
     plt.colorbar(im1, cax=cax)
     plt.tight_layout()
-    plt.savefig(saveFilename)
+    if saveFilename:
```

Comment:
I did this because I wanted to be agnostic between this and `''`. I'm never sure which to put for the default "don't save" option either, but that's why I went for falsey rather than `is not None`.